core/silkworm/common/util.cpp

```diff
@@ -83,20 +83,14 @@ evmc::bytes32 to_bytes32(ByteView bytes) {
     return out;
 }
 
-ByteView zeroless_view(const ByteView& data) {
+ByteView zeroless_view(ByteView data) {
     unsigned offset{0};
     while (offset < data.length() && data[offset] == 0) {
         ++offset;
     }
     return {data.data() + offset, data.length() - offset};
```

Comment:
Probably more basic_string_view style to return `data.substr(offset)`

Reply:
Right, makes sense